### PR TITLE
adding files to repo

### PR DESCRIPTION
**Pull request by Danelif**
This is my first PR for Hello Microverse setup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,22 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
```
Comment (zieeco) [inline README.md:5]: **[REQUIRED CHANGES]**

- Please, kindly add a description about the project to make your repo appear more professional.

File: README.md [hunk 2]
```diff
@@ -1,73 +1,22 @@
...
-
+# Hello-microverse
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
```
Comment (zieeco) [inline README.md:13]: **[REQUIRED CHANGES]**

- Please, kindly replace the `Major languages` with `HTML`, and `Frameworks` with `CSS`, and remove the `Technologies` to make the readme well arranged to look more professional.

File: README.md [hunk 3]
```diff
@@ -1,73 +1,22 @@
...
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+html and css
```
Comment (zieeco) [inline README.md:7]: - Please, kindly remove this from here as you've updated it in the previous review as I've required the changes.

File: README.md [hunk 4]
```diff
@@ -1,73 +1,22 @@
...
 
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```
Comment (zieeco) [inline README.md]: **[REQUIRED CHANGES]**

- Please, kindly remove this text content as they're not descriptive in telling the user how to have your project running locally on their machine. Instead, kindly add some instructions telling the users how to have your repo. follow the steps below:

To get a local copy up and running follow these simple example steps.

1. In the `Getting started` header, you should add some instructions on how a user can have your repo locally on their machine. (how to clone the repo, how to move into the cloned repo), 
2. Prerequisite: e.g `vscode editor`, a `web browser(chrome, etc)`.

File: README.md [hunk 5]
```diff
@@ -1,73 +1,22 @@
...
-### Usage
-
-### Run tests
-
-### Deployment
-
-
-
-## Authors
-
```
Comment (zieeco) [inline README.md]: - Please, kindly add back this authors header but now you should make it singular as you've worked on this project solely. After updating the author header, rename author1 to author.

File: README.md [hunk 6]
```diff
@@ -1,73 +1,22 @@
...
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
```
Comment (zieeco) [inline README.md:73]: - Please, kindly update this section of your readme by doing the following:

1. Contribution header. Kindly add all the contents in the contributing header, as you've removed some contents from it. You can refer to the [microverse readme tamplate](https://github.com/microverseinc/readme-template/blob/master/README.md) to add the missing contents.
2. Show your support header.
3. Acknowledgments.
4. License.

:point_right: The arrangement should follow that used in the [microverse readme tamplate](https://github.com/microverseinc/readme-template/blob/master/README.md) to keep consistency to make your repo appear professional.

File: .gitignore [hunk 7]
```diff
@@ -0,0 +1,3 @@
+node_modules
+package-lock.json
```
Comment (zieeco) [inline .gitignore]: **[REQUIRED CHANGES]**

- Please, kindly remove the package-lock.json file from the gitignore as this needs to be visible in your repository.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi, @Danelif, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Great start :+1: 
- Descriptive PR :+1: 
- No Linter error :+1: 


## Required Changes ♻️

- [ ] Please, kindly add a description to your PR by clicking the three dotted horizontal line as seen in the screenshot image below:

description



_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 7 above)
Comment (roshan-bajgain) [inline .gitignore]: - [ ] Kindly remove this `package-lock.json`  from .gitignore as requested by previous reviewer

File: README.md [hunk 8]
```diff
@@ -1,73 +1,24 @@
...
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+html and css
 
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```
Comment (roshan-bajgain) [inline README.md]: - [ ] Kindly remove this and add a Getting started section like how to clone the repo and who to see on the browser
- [ ] Prerequisite: e.g vscode editor, a web browser(chrome, etc).

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Danelif, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @Danelif , 
Good job so far! 👏🏼 @

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed :heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks good:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 


 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Danelif** [comment]
PR accepted

**Pull request merged**